frontend/src/DummyRootView.ts

```diff
@@ -57,11 +57,6 @@ export default defineComponent({
         // - /sql-editor if defaultWorkspaceView == 'EDITOR'
         // - lastVisit.path if not empty
         // - /issues as fallback
-        // Only turned on when isDev() is true
-        if (!isDev()) {
```

Comment:
I found we never release "redirect to last visit" feature in the prod...How about release it in this PR?

Reply:
Let's do it.